Rodents/Portal_rodent.csv

```diff
@@ -76352,3 +76352,232 @@
+76567,7,19,2023,520,3,,35,"PP","M","Z",,,,,,,22,22,"05673D",,,,,,,,,,,
+76568,7,19,2023,520,3,,32,"PP","F","Z",,,,,,,22,12,"C27953",,,,,,,,,,,
+76569,7,19,2023,520,3,,21,"PB","F","Z",,,,,,,25,25,"795E9B","*",,,,,,,,,,
+76570,7,19,2023,520,3,,11,"PP","F","Z",,,,,,,22,16,"056703",,,,,,,,,,,
+76571,7,19,2023,520,3,3,17,"PP","M","Z",,,,,,,21,17,"BFAEFB",,,,,,,,,,,
+76572,7,19,2023,520,3,,27,"PP","F",,"J",,,,,,21,9,"79A03B","*",,,,,,,,,,
+76573,7,19,2023,520,3,,25,"PP","F",,,,,,"E",,22,16,"79A055",,,,,,,,,,,
+76574,7,19,2023,520,17,,71,"PP","F","Z",,,,,,,21,16,"78B045",,,,,,,,,,,
+76575,7,19,2023,520,17,,74,"DM","M",,,"S",,,,,37,48,"BF1125",,,,,,,,,,,
+76576,7,19,2023,520,17,,31,"DM","F",,,,,"P","B",,37,52,"C2B4A7",,,,,,,,,,,
+76577,7,19,2023,520,17,,72,"PP","F","Z",,,,,,,20,11,"7A0406","*",,,,,,,,,,
+76578,7,19,2023,520,17,,61,"DM","F","Z",,,,,,,35,40,"05C448",,,,,,,,,,,
+76579,7,19,2023,520,17,,23,"DM","M","Z",,,,,,,35,40,"05C383",,,,,,,,,,,
+76580,7,19,2023,520,17,,17,"DM","M",,,"S",,,,,35,45,"799F37",,,,,,,,,,,
+76581,7,19,2023,520,17,,41,"PP","F",,,,,,"E",,20,16,"BFD9CF",,,,,,,,,,,
+76582,7,19,2023,520,17,,26,"DM","F",,,,,,"E",,35,43,"C2B0A1",,,,,,,,,,,
+76583,7,19,2023,520,17,,52,"PP","F","Z",,,,,,,21,15,"78BFB0",,,,,,,,,,,
+76584,7,19,2023,520,17,,15,"OT","F","Z",,,,,,,17,21,"05C497",,,,,,,,,,,
+76585,7,19,2023,520,17,,56,"PP","F",,"J",,,,,,21,9,"78AE3C","*",,,,,,,,,,
+76586,7,19,2023,520,17,,57,"PP","F","Z",,,,,,,21,14,"7961B6","*",,,,,,,,,,
+76587,7,19,2023,520,17,,37,"PP","F",,"J",,,,,,21,10,"78AEFE","*",,,,,,,,,,
+76588,7,19,2023,520,15,,74,"PP","F",,,,,,,"L",21,16,"793E71",,,,,,,,,,,
+76589,7,19,2023,520,15,,71,"PP","F","Z",,,,,,,22,13,"C279EB","*",,,,,,,,,,
+76590,7,19,2023,520,15,,72,"PP","F","Z",,,,,,,20,15,"C2B4DE",,,,,,,,,,,
+76591,7,19,2023,520,15,,24,"PP","F",,,,,,"E",,22,19,"056752",,,,,,,,,,,
+76592,7,19,2023,520,15,,37,"OT","F","Z",,,,,,,21,20,"79C73D","*",,,,,,,,,,
+76593,7,19,2023,520,15,,76,"PP","F","Z",,,,,,,21,17,"78BDD8",,,,,,,,,,,
+76594,7,19,2023,520,15,,73,"PP","M",,,"M",,,,,20,20,"C2BD3C",,,,,,,,,,,
+76595,7,19,2023,520,15,,27,"PP","M","Z",,,,,,,21,16,"78AF64",,,,,,,,,,,
+76596,7,19,2023,520,15,,33,"PP","F",,,,,,"E",,20,16,"0565EA",,,,,,,,,,,
+76597,7,19,2023,520,15,,26,"PP","M","Z",,,,,,,21,19,"78BE38",,,,,,,,,,,
+76598,7,19,2023,520,15,,34,"PP","F","Z",,,,,,,21,16,"05C45B",,,,,,,,,,,
+76599,7,19,2023,520,15,,17,"PP","F","Z",,,,,,,20,14,"79FC0D",,,,,,,,,,,
+76600,7,19,2023,520,15,,36,"PP","F","Z",,,,,,,21,16,"05BC4B",,,,,,,,,,,
+76601,7,19,2023,520,15,,63,"PP","F",,,,,,"E",,20,14,"C2E26E",,,,,,,,,,,
+76602,7,19,2023,520,15,3,43,"OT","M",,,"R",,,,,19,20,"79C5C1","*",,,,,,,,,,
+76603,7,19,2023,520,14,,65,"DM","F","Z",,,,,,,33,41,"7A0433",,,,,,,,,,,
+76604,7,19,2023,520,14,,73,"PP","F","Z",,,,,,,21,16,"056658",,,,,,,,,,,
+76605,7,19,2023,520,14,,72,"PP","F","Z",,,,,,,21,14,"056749",,,,,,,,,,,
+76606,7,19,2023,520,14,,49,"PP","M","Z",,,,,,,21,19,"C2BA62",,,,,,,,,,,
```

Comment:
Looks like this is the faulty stake. How'd it make it past the cleaning checks?

Reply:
aakk! sorry. I'm not sure. I only got an error for suspect stakes that were repeated but not this. I'll fix it!